.circleci/config.yml

```diff
@@ -110,6 +110,16 @@ jobs:
     steps:
       - run: {command: echo "Ran all required tests successfully"}
 
+  trigger-antithesis-simulation:
+    machine:
+      image: ubuntu-2204:2023.10.1
```

Comment:
Note: This is different to

```
  test-suite:
    parameters:
      suite:
        type: integer
    machine:
      image: ubuntu-2004:202111-02
```

Was there a specific reason you chose this version, or just because it's newer? We probably want to bump the other image eventually too, but it seems like they're not automatically bumped - in which case, we'll need to set a task to bump these periodically, or write an excavator.

Reply:
Yeah, just got the newer image, because why not. But happy to continue using the old one if we want to centralize in a single PR or excavator whenever we change it. Let me know what you prefer.

Reply:
Nah, I prefer the newer one. But we should have some way of keeping both up to date in the medium term. Mind tracking that on our support backlog tracking tool?